test/benchmarks/ramunajan_pi.inputs

```diff
@@ -0,0 +1,14 @@
+pi_10_runs
+ramunajan
+i:I
+10
+
+31415926535897932
+
+pi_30_runs
```

Comment:
I don't think adding more than one execution benchmark adds any additional information. It rather just make running benchmarks take longer.